components/options/Options.tsx

```diff
@@ -0,0 +1,130 @@
+import Sidebar from "components/sidebar/Sidebar";
+import { FC, useState, useEffect, useRef, FormEvent } from "react";
+import React from "react";
+import { IUser } from "../../types/User";
+import UserItem from "../user/UserItem";
+import axios from "node_modules/axios/index";
+import ConfirmDialog from "components/dialog/ConfirmDialog";
+import OptionsAPI from "api/options";
+
+  interface Props {}
+
+  const Options: FC<Props> = ({}) => {
+    const nameRef = useRef<HTMLInputElement>();
+    const pwRef = useRef<HTMLInputElement>();
+    const email = "foo@mail.de"
+    const [user, setUsers] = useState({username:"foo", password:"12345"});
```

Comment:
die Variablenname und der Setter sollten idealerweise gleich heissen
```suggestion
    const [users, setUsers] = useState<IUser>({username:"foo", password:"12345"});
```